cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/elasticsearch/ElasticsearchBackendImpl.java

```diff
@@ -117,10 +117,10 @@ public JsonResponse bulkInsert(String index, String type, List<Map<String, Strin
                 jsonLines += String.format("{\"index\":{\"_id\":\"%s-%s\"}}\n", erecvTimeTs, hash);
                 jsonLines += String.format("%s\n", edata);
             } // for
-            entity = new StringEntity(jsonLines);
+            entity = new StringEntity(jsonLines, Charset.forName("UTF-8"));
```

Comment:
"UTF-8" should be declared as a String, so that if anytime another content-type is supported it would be easyer to modify it.

Reply:
@IvanHdzC Thank you for your advice. That's right, I think. I'll try to get 'content_type' as a configuration parameter.

Reply:
Great! Please don't forget to update your branch.